.github/workflows/ci.yml

```diff
@@ -36,6 +36,16 @@ jobs:
         key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
         restore-keys: ${{ runner.os }}-m2
 
+    - name: Checkout OpenSearch
+      uses: actions/checkout@v2
+      with:
+        repository: 'opensearch-project/OpenSearch'
```

Comment:
was opendistro-security already removed from this file?

Reply:
It is a newly added task to checkout OpenSearch repo in order to build OpenSearch artifacts in the following step before buliding security plugin on top of it given that OpenSearch artifacts hasn't been available in Maven repo as of now. Security plugin is built in the `Package` step and didn't get removed.